src/_gettsim_tests/test_interface.py

```diff
@@ -685,6 +686,24 @@ def test_fail_if_cannot_be_converted_to_internal_type(
         convert_series_to_internal_type(input_data, expected_type)
 
 
+@pytest.mark.parametrize(
+    "data, functions_overridden",
+    [
+        (
+            pd.DataFrame({"bg_id": [1, 2, 3]}),
+            {"bg_id": bg_id_numpy},
+        ),
+        (
+            pd.DataFrame({"wthh_id": [1, 2, 3]}),
+            {"wthh_id": wthh_id_numpy},
+        ),
+    ],
+)
+def test_user_provided_groupings(data, functions_overridden):
```

Comment:
```suggestion
def test_provide_groupings(data, functions_overridden):
```
Else one may think that this does some consistency checks, but it is really only about whether we handle that case at all, right?